src/Avalonia.Base/Media/Imaging/RenderTargetBitmap.cs

```diff
@@ -62,10 +62,16 @@ private static IRenderTargetBitmapImpl CreateImpl(PixelSize size, Vector dpi)
             return factory.CreateRenderTargetBitmap(size, dpi);
         }
 
-        public DrawingContext CreateDrawingContext()
+        /// <summary>
+        /// Creates a <see cref="DrawingContext"/> for drawing to the <see cref="RenderTargetBitmap"/>.
+        /// </summary>
+        /// <param name="clear">Indicates if the image should be cleared.</param>
+        /// <returns>The drawing context.</returns>
+        public DrawingContext CreateDrawingContext(bool clear = true)
```

Comment:
This is a binary-breaking change, and we can't accept it in 11.x timeline.
If you want to add this parameter, it should be a new method that accepts it.
cc @kekekeks if this change makes sense with planned RTB refactorings.

Reply:
The general plan is to have a separate class for compositor-driven rendering, so I guess we could extend RTB this way.

Reply:
I moved the behavior to a new function named CreateEditDrawingContext() and removed the optional parameter from CreateDrawingContext().